src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+  padding: 32px 16px 17px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    align-self: center;
+  }
+
+  &__title {
+    padding: 40px 0 4px;
```

Comment:
it is better to use margin for setting spasing between elements.
and be consistent use either margin top or margin bottom everywhere

Reply:
not fixed